scripts/build-packages.py

```diff
@@ -9,14 +9,14 @@
 import nose
 
 PYTHON_VERSIONS = ["27", "34"]
+CONDA_NPY = "19"
 
 
 def build_recipe(recipe):
     try:
-        os.environ["CONDA_NPY"] = "18"
         for py in PYTHON_VERSIONS:
-            os.environ["CONDA_PY"] = py
             sp.check_output(["conda", "build", "--no-anaconda-upload",
+                             "--numpy", CONDA_NPY, "--python", py,
                              "--skip-existing", "--quiet", recipe],
                             stderr=sp.STDOUT)
     except sp.CalledProcessError as e:
```

Comment:
You can replicate the following line after the check_output call. This will ensure that output is shown in nose verbose mode also if no error is thrown.